css/index.css

```diff
@@ -143,4 +143,158 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* sprint challenge
+
+html {
+    font-size:62.5%;
+}
+*/
+
+
+a{
+    text-decoration: none;
+    color: black;
+}
+
+/* Desktops and laptops ----------- */
+@media only screen  and (min-width : 1000px) and (max-width : 2000px) {
```

Comment:
You can omit `only screen and (min-width)` and still get the same effect, with your other media queries override this one when the device width gets smaller.

Also, you're having to repeat your code being too specific. I.e.:

```
.middle-content .boxes .box:nth-of-type(2) {
    background-color: forestgreen;       
        }

        .middle-content .boxes .box:nth-of-type(4) {
        background-color: darkorchid;       
    }

    .middle-content .boxes .box:nth-of-type(6) {
        background-color: hotpink;       
    }

    .middle-content .boxes .box:nth-of-type(8) {
        background-color: indigo;       
    }

    .middle-content .boxes .box:nth-of-type(10) {
        background-color: lawngreen;       
    }
```